plugin/src/test/groovy/org/gradle/testretry/testframework/TestNGFuncTest.groovy

```diff
@@ -60,7 +60,7 @@ class TestNGFuncTest extends AbstractFrameworkFuncTest {
         with(result.output) {
             it.count('lifecycle FAILED') == 1
             it.count('lifecycle PASSED') == 1
-            !it.contains("org.gradle.test-retry was unable to retry")
+            !it.contains("The following test methods could not be retried")
```

Comment:
💬 No similar test for TestNG, since in this case, it will still report descriptors for failed but not re-tried methods. They will be marked as skipped.